### PR TITLE
fixed problems associated with tests

### PR DESCRIPTION
**Pull request by lagartoverde**
This PR makes the tests on #42 pass

### CODE REVIEW

File: support/templates/behandeling-html.hbs [hunk 1]
```diff
@@ -77,7 +69,7 @@
         <p>
           Met als gevolg,
         </p>
-        <p property="besluit:gevolg">{{this.result}}</p>
+        <p datatype="rdf:langString" property="besluit:gevolg">{{this.result}}</p>
```
Comment (nvdk) [inline support/templates/behandeling-html.hbs:72]: this is not the correct solution, for it to be a valid langstring we should set the language of the the value. the datatype is then automatically a langstring. See https://www.w3.org/TR/html-rdfa/#h-specifying-the-language-for-a-literal

**Review by nvdk — CHANGES_REQUESTED** [review]
small issue wrt langstring, other than that looks good!

**Comment by nvdk** [comment]
could it be this is missing the changes to the shacl file that we discussed?

File: support/templates/behandeling-html.hbs (hunk 1 above)
Comment (lagartoverde) [reply support/templates/behandeling-html.hbs:72]: Yes, you are right I'm missing the language, I was going to add it but I didn't, are you sure we don't need the datatype for marawa to pick it up?

**Comment by lagartoverde** [comment]
What shacl changes? I changed the base branch because I don't know why it was set to the other one, the shacl changes are in the other branch that is the one with the tests, this is only for the fixes

File: support/templates/behandeling-html.hbs (hunk 1 above)
Comment (nvdk) [reply support/templates/behandeling-html.hbs:72]: hopefully marawa does the right thing

**Comment by nvdk** [comment]
> What shacl changes? I changed the base branch because I don't know why it was set to the other one, the shacl changes are in the other branch that is the one with the tests, this is only for the fixes

indeed that comments was meant for the other pr :)

File: support/templates/behandeling-html.hbs (hunk 1 above)
Comment (nvdk) [reply support/templates/behandeling-html.hbs:72]: I've added a PR on marawa so it does the right thing, this will be a good testcase for it :)

**Pull request merged**